www/archive.inc

```diff
@@ -3,9 +3,20 @@ require_once('testStatus.inc');
 require_once('logging.inc');
 require_once('common_lib.inc');
 
+function S3_GetSdkClient() {
+    require_once('./lib/S3.php');

```

Comment:
I don't think the s3 library that is being used supports IAM roles.  It only takes credentials explicitly.  The s3 archiving probably needs to be moved to use the official AWS SDK (like the EC2 code does) and fall back to the s3 lib for Internet Archive storage.